spec/valvat/lookup_spec.rb

```diff
@@ -4,25 +4,54 @@
 
 describe Valvat::Lookup do
   describe '#validate' do
-    context 'with existing VAT number' do
+    context 'with existing EU VAT number' do
       it 'returns true' do
         result = described_class.validate('IE6388047V')
-        # Skip if VIES is down
-        expect(result).to be(true) unless result.nil?
+        skip "VIES is down" if result.nil?
+        expect(result).to be(true)
```

Comment:
I changes a couple of these conditional to use the standard `skip` method from rspec this way when a test a skipped for this reason (service down) it actually shows in the test output, so the test is valid but you can see that it was skipped.

Reply:
@yolk I see you applied this in your `nethttp` branch so I'll mark this as resolved